geogem.rb

```diff
@@ -35,13 +35,13 @@ def centre_point
 
   def area
     multiplied = lines.map { |p1, p2| (p1.x * p2.y) - (p2.x * p1.y) }
```

Comment:
Minor comment but 'multiplied' isn't very descriptive

Reply:
Good point. That's from the core code but we should probably change it.